reframe/core/logging.py

```diff
@@ -386,20 +387,30 @@ class HTTPJSONHandler(logging.Handler):
         'stack_info', 'thread', 'threadName', 'exc_text'
     }
 
-    def __init__(self, url, extras=None):
+    def __init__(self, url, extras=None, perflog_ignore=None):
         super().__init__()
         self._url = url
         self._extras = extras
+        self._perflog_ignore = perflog_ignore
 
     def _record_to_json(self, record):
+        def _log_record(key):
+            if key.startswith('_') or key in HTTPJSONHandler.LOG_ATTRS:
+                return False
+
+            if self._perflog_ignore and key in self._perflog_ignore:
+                return False
```

Comment:
Why don't you combine this in a single if?

Reply:
I thought it might be more readable but you are right, I changed it.